app/src/main/res/menu/menu_item_creating_conversation_no_preview.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
WARNING: [UnusedResources] The resource `R.menu.menu_item_creating_conversation_no_preview` appears to be unused
 Unused resources make applications larger and slow down builds.